components/movementsensor/utils.go

```diff
@@ -134,3 +160,16 @@ func (le *LastError) Get() error {
 	le.count = 0
 	return errToReturn
 }
+
+// ArePointsEqual checks if two geo.Point instances are equal.
+func (lp *LastPosition) ArePointsEqual(p1, p2 *geo.Point) bool {
```

Comment:
Are we at the point were we need to write tests for these functions in utils? If so, let's start doing that.

Reply:
Follow up ticket then, and add it to epic.